integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/iud/UpdateCarbonTableTestCase.scala

```diff
@@ -788,7 +788,20 @@ class UpdateCarbonTableTestCase extends QueryTest with BeforeAndAfterAll {
     sql("""drop table if exists iud.dest33_part""")
   }
 
-    override def afterAll {
+  test("[CARBONDATA-3477] deal line break chars correctly after 'select' in 'update ... select columns' sql") {
+    sql("""drop table if exists iud.dest11""").show
+    sql("""create table iud.dest11 (c1 string,c2 int,c3 string,c5 string) STORED BY 'org.apache.carbondata.format'""")
+    sql(s"""LOAD DATA LOCAL INPATH '$resourcesPath/IUD/dest.csv' INTO table iud.dest11""")
+    sql("""update iud.dest11 d set (d.c3, d.c5 ) = (select
```

Comment:
can you add more testcase, like adding some blank character after the `select`, for example:
"update iud.dest d set (d.c3, d.c5) = (select    \n s.c33 ... "
"update iud.dest d set (d.c3, d.c5) = (select\t\n s.c33 ... "

Reply:
Done